vite.config.ts

```diff
@@ -1,16 +1,33 @@
 import { resolve } from "path";
 import { defineConfig } from "vitest/config";
 import dts from "vite-plugin-dts";
+import packageJson from "./package.json";
 
 export default defineConfig({
```

Comment:
🟢 Let's raise a ticket to back-port these changes into the template

Reply:
Will do

Reply:
https://github.com/lokalise/npm-package-template/pull/32/files